imap_processing/ialirt/l0/parse_mag.py

```diff
@@ -568,6 +568,19 @@ def process_packet(
     pkt_counter = get_pkt_counter(accumulated_data["mag_status"])
     accumulated_data["pkt_counter"] = pkt_counter
 
+    # Convert from incrementing uint16 (0-65535) to radians.
+    sc_spin_phase_rad = accumulated_data["sc_spin_phase"].astype(float) * (
+        2 * np.pi / 65535.0
+    )
+    sc_inertial_right = accumulated_data["sc_inertial_right"].astype(float) * (
+        2 * np.pi / 65535.0
+    )
+    sc_inertial_decline = (
+        accumulated_data["sc_inertial_decline"].astype(float) / 65535.0
```

Comment:
Rename to declination? I don't know that decline is the right wording.
```suggestion
        accumulated_data["sc_inertial_declination"].astype(float) / 65535.0
```